pipelines/build/regeneration/jdk10_regeneration_pipeline.groovy

```diff
@@ -12,16 +12,25 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-String javaVersion = "jdk10u"
+String javaVersion = "jdk10"
 
 node ("master") {
   try {
     def scmVars = checkout scm
     load "${WORKSPACE}/pipelines/build/common/import_lib.groovy"
   
-    def buildConfigurations = load "${WORKSPACE}/pipelines/jobs/configurations/${javaVersion}_pipeline_config.groovy"
-
-    println "[INFO] Found buildConfigurations:\n$buildConfigurations"
+    // Load buildConfigurations from config file. This is what the nightlies & releases use to setup their downstream jobs
+    def buildConfigurations = null
+    def buildConfigPath = "${WORKSPACE}/pipelines/jobs/configurations/${javaVersion}_pipeline_config.groovy"
+
+    if (fileExists(buildConfigPath)) {
+      buildConfigurations = load buildConfigPath
+    } else {
+      javaVersion = javaVersion + "u"
+      println "[WARNING] ${buildConfigPath} does not exist.\n[INFO] Trying ${WORKSPACE}/pipelines/jobs/configurations/${javaVersion}_pipeline_config.groovy..."
```

Comment:
So I'm thinking that `u` is the common case here and I'm not sure we want to print a WARNING message in each case.  Perhaps that should be dropped to an INFO and the language changed, for example:

`[INFO] ${buildConfigPath} does not exist, chances are we want a ${javaVersion} repo.\n[INFO] Trying ${WORKSPACE}/pipelines/jobs/configurations/${javaVersion}_pipeline_config.groovy...